.ci/install_kata_kernel.sh

```diff
@@ -73,20 +76,14 @@ build_and_install_kernel() {
 	popd >> /dev/null
 }
 
-install_packaged_kernel(){
-	info "Install packaged kernel version"
-	rc=0
-	if [ "$ID"  == "ubuntu" ] || [ "$ID" == "debian" ]; then
-		chronic sudo apt install -y "$packaged_kernel" || rc=1
-	elif [ "$ID"  == "fedora" ]; then
-		chronic sudo dnf install -y "$packaged_kernel" || rc=1
-	elif [ "$ID"  == "centos" ]; then
-		chronic sudo yum install -y "$packaged_kernel" || rc=1
-	else
-		die "Unrecognized distro"
-	fi
-
-	return "$rc"
+install_prebuilt_kernel(){
+	info "Install pre-built kernel version"
+	local kernel_version=$(curl -qL "${latest_build_url}/latest")
+	mkdir -p "${kernel_dir}"
+	local kernel_binary_name="vmlinuz-${kernel_version}"
+	local kernel_binary_path="${kernel_dir}/${kernel_binary_name}"
+	sudo -E curl -L "${latest_build_url}/${kernel_binary_name}" -o "${kernel_binary_path}"
```

Comment:
maybe wrt the fc CI fail - do we need to build and pull down the vmlinux as well as the vmlinuz?